### ISSUE TITLE
torch.nn.LogSoftmax.__repr__() does not include dim argument

### ISSUE THREAD

**Issue by SdgJlbl** [pytorch/pytorch#19961]
## 🐛 Bug

The `__repr__` of the class torch.nn.LogSoftmax does not contain the value of the dim argument, even if the implicit dimension choice for softmax has been deprecated. 

## To Reproduce

```
import torch.nn as nn
nn.LogSoftmax(dim=1).__repr__()
```
Output
```
'LogSoftmax()'
```
Expected
```
'LogSoftmax(dim=1)'
```

## Expected behavior

I would have expected the same behaviour as from classes such as torch.nn.Conv2d or torch.nn.MaxPool2d.

## Environment

 - PyTorch Version (e.g., 1.0): 1.0.1.post2
 - OS (e.g., Linux): Ubuntu 18.04.2 LTS
 - How you installed PyTorch (`conda`, `pip`, source): conda
 - Build command you used (if compiling from source):
 - Python version: 3.6.8

**Comment by SdgJlbl** [comment]
I'm happy to submit a PR if that helps :)

**Comment by apaszke** [comment]
Please do!

**Comment by Stonesjtu** [comment]
@SdgJlbl  reminder that you can override `extra_repr` instead of  `__repr__`.
